langchain/src/vectorstores/qdrant.ts

```diff
@@ -84,13 +84,18 @@ export class QdrantVectorStore extends VectorStore {
    * from the documents using the `Embeddings` instance and then adds the
    * vectors to the database.
    * @param documents Array of `Document` instances to be added to the Qdrant database.
+   * @param customPayload Optional 'object' in JSON format used to query database on more criteria
    * @returns Promise that resolves when the documents have been added to the database.
    */
-  async addDocuments(documents: Document[]): Promise<void> {
+  async addDocuments(
+    documents: Document[],
+    customPayload?: object[]
```

Comment:
Can you change this to an object with a field called `customPayload`? That better matches the base interface:

https://github.com/langchain-ai/langchainjs/blob/main/langchain-core/src/vectorstores.ts#L126